pkgs/applications/graphics/mandelbulber2/default.nix

```
@@ -0,0 +1,38 @@
+{ lib, mkDerivation, fetchFromGitHub, libpng, gsl, libsndfile, lzo,
```

```suggestion
{ lib
, mkDerivation
, fetchFromGitHub
, libpng
, gsl
, libsndfile
, lzo,
```

is more diff-friendly. See nixpkgs-fmt.

---

Done, used nixpkgs-fmt.